src/main/kotlin/org/openbroker/no/mortgage/model/Applicant.kt

```diff
@@ -0,0 +1,69 @@
+package org.openbroker.no.mortgage.model
+
+import org.openbroker.common.obfuscateDigits
+import org.openbroker.common.requireMin
+import org.openbroker.no.model.Address
+import org.openbroker.no.model.EmploymentStatus
+import org.openbroker.no.model.HousingType
+import org.openbroker.no.model.MaritalStatus
+
+data class Applicant @JvmOverloads constructor(
+    val ssn: String,
+    val phone: String? = null,
+    val secondaryPhone: List<String> = emptyList(),
+    val emailAddress: String? = null,
+    val employmentStatus: EmploymentStatus,
+    val employmentStatusSinceYear: Int,
+    val employmentStatusSinceMonth: Int,
+    val employerName: String? = null,
+    val employerPhone: String? = null,
+    val dependentChildren: Int,
+    val childSupportReceivedMonthly: Int? = null,
+    val rentReceivedMonthly: Int? = null,
+    val otherIncomeReceivedMonthly: Int? = null,
+    val childSupportPaidMonthly: Int? = null,
+    val paymentRemark: Boolean,
+    val housingType: HousingType,
+    val housingCostPerMonth: Int,
+    val monthlyGrossIncome: Int,
```

Comment:
One creditor really needed monthlyGrossIncome, and since we already have monthlyNetIncome, it seemed better to have both as monthly (after all we can just multiply with 12 to get yearly so).

Reply:
Ah, then it feels like separate use cases so yeah, no need to sync.